packages/create-instantsearch-app/package.json

```diff
@@ -41,9 +41,9 @@
     "latest-semver": "2.0.0",
     "load-json-file": "6.2.0",
     "lodash.camelcase": "4.3.0",
-    "metalsmith": "2.3.0",
-    "metalsmith-ignore": "1.0.0",
-    "metalsmith-in-place": "4.4.1",
+    "metalsmith": "2.5.1",
+    "@metalsmith/remove": "1.3.0",
+    "@metalsmith/in-place": "4.6.0",
```

Comment:
Will those versions match those in the helper?

Reply:
the helper already used different versions for the site, but mostly the same, yes